### PR TITLE
Zoom sdk update

### PR DESCRIPTION
**Pull request by NavayuvanSB**
### Changes done
- Add support for git lfs
- Update zoom sdk to v5.5.1.1319

### Reason for the changes
- Previously using zoom sdk is depreciated

### CODE REVIEW

File: mobilertc/build.gradle [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+
+dependencies.add("default","androidx.security:security-crypto:1.1.0-alpha02")
+dependencies.add("default","com.google.crypto.tink:tink-android:1.5.0")
+dependencies.add("default","androidx.swiperefreshlayout:swiperefreshlayout:1.0.0")
+
+dependencies.add("default","androidx.appcompat:appcompat:1.0.0")
+dependencies.add("default","androidx.constraintlayout:constraintlayout:1.1.3")
+dependencies.add("default","com.google.android.material:material:1.2.0-alpha03")
+dependencies.add("default","com.google.android:flexbox:2.0.1")
+dependencies.add("default","androidx.multidex:multidex:2.0.0")
```
Comment (Karthik-0) [inline mobilertc/build.gradle:12]: Why are these added

**Review by Karthik-0 — CHANGES_REQUESTED** [review]

File: mobilertc/build.gradle (hunk 1 above)
Comment (NavayuvanSB) [reply mobilertc/build.gradle:12]: It's added automatically with new sdk version.

Comment (NavayuvanSB) [reply mobilertc/build.gradle:12]: It's added automatically with new sdk version.

Comment (NavayuvanSB) [reply mobilertc/build.gradle:12]: It's added automatically with new sdk version.

Comment (NavayuvanSB) [reply mobilertc/build.gradle:12]: It's added automatically with new sdk version.

**Review by Karthik-0 — APPROVED** [review]
